Demo/Sources/Search/SearchView.swift

```diff
@@ -51,12 +52,27 @@ struct SearchView: View {
                         model.loadMore()
                     }
                 }
-#if os(iOS)
                 .swipeActions { CopyButton(text: media.urn) }
-#endif
+                .scrollDismissesKeyboard(.immediately)
+                .refreshable { await model.refresh() }
             }
-            .scrollDismissesKeyboard(.immediately)
-            .refreshable { await model.refresh() }
+#else
+            ScrollView {
+                ForEach(medias, id: \.urn) { media in
+                    let title = MediaDescription.title(for: media)
+                    Cell(title: title, subtitle: MediaDescription.subtitle(for: media), style: MediaDescription.style(for: media)) {
+                        let media = Media(title: media.title, type: .urn(media.urn))
+                        router.presented = .player(media: media)
+                    }
+                    .padding(.horizontal, 50)
+                    .onAppear {
+                        if let index = medias.firstIndex(of: media), medias.count - index < kPageSize {
+                            model.loadMore()
+                        }
+                    }
```

Comment:
How about creating a dedicated method for that?
```swift
@ViewBuilder
private func content(_ media: SRGMedia, _ medias: [SRGMedia]) -> some View {
    let title = MediaDescription.title(for: media)
    Cell(title: title, subtitle: MediaDescription.subtitle(for: media), style: MediaDescription.style(for: media)) {
        let media = Media(title: media.title, type: .urn(media.urn))
        router.presented = .player(media: media)
    }
#if !os(iOS)
    .padding(.horizontal, 50)
#endif
    .onAppear {
        if let index = medias.firstIndex(of: media), medias.count - index < kPageSize {
            model.loadMore()
        }
    }
#if os(iOS)
    .swipeActions { CopyButton(text: media.urn) }
    .scrollDismissesKeyboard(.immediately)
    .refreshable { await model.refresh() }
#endif
}
```

And the `!media.isEmpty` block of `loadedView` becomes:
```swift
#if os(iOS)
List(medias, id: \.urn) { media in
    content(media, medias)
}
#else
ScrollView {
    ForEach(medias, id: \.urn) { media in
        content(media, medias)
    }
}
#endif
```

Reply:
Done!